Homework_2211/test_town.py

```diff
@@ -0,0 +1,124 @@
+"""Testing buildWorker.py."""
+import buildWorker
+import pytest
+from json import load
+
+building_ptrs = [('Hello', 7, 6, 1, 2, 3), ('Second', 1.8, 3.2, 4, 1, 3)]
+
+
+@pytest.mark.parametrize('name, height, width, floors, coord_x, coord_y', building_ptrs)
+def test_building_ptrs(name, height, width, floors, coord_x, coord_y) -> None:
+    """Test for param of elements of class Building.
+
+    Args:
+        name: str - name of a building.
+        height: float - height of a building.
+        width: float - width of a building.
+        floors: int - number of floors of a building.
+        coord_x: int - coordinate x of a building on the map.
+        coord_y: int - coordinate y of a building on the map.
+    """
+    assert buildWorker.Building(name, height, width, floors, coord_x, coord_y).name == name
```

Comment:
:crying_cat_face: 
building = buildWorker.Building(name, height, width, floors, coord_x, coord_y)
for attr, value in tests: assert getattr(building, attr) == value

:running_man: